python/metatensor-torch/metatensor/torch/atomistic/ase_calculator.py

```diff
@@ -66,10 +66,20 @@ class MetatensorCalculator(ase.calculators.calculator.Calculator):
     installed, in which case it will be used instead.
     """
 
+    additional_outputs: Dict[str, TensorMap]
```

Comment:
```suggestion
    additional_outputs: Dict[str, TensorMap] = {}
```
Should this be initialized? Perhaps to `{name: None for name in ...}`

Reply:
I don't see it being initialized in the `__init__`, maybe I'm being dumb